### DISCUSSION TITLE
How to connect the ios、andriod application to skywalking

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8745] [Q&A]
1. How to filter out unnecessary micro service interfaces in the dashboard;
2. How to display HTTP parameters, function and method names and other information on the tracking page;
3. How to configure the log collection;
4. How to view the whole link according to the user token;
5. How to connect the ios、andriod application to skywalking;
6. In the skywalgking tracking page, how to filter out the output information such as context connection and display only key information to reduce interference

**Comment by wu-sheng** [comment]
Most are documented, read documents first.

**Comment by rexconquer** [comment]
How to connect the ios、andriod application to skywalking? I can't find any sdk about ios and android.

**Reply by wu-sheng** [reply]
No, there's not. No one working around ios things connected with us.

**Reply by rexconquer** [reply]
Thx for respose. What I think is how to trace the frontend access with skywalking. We have tingyun now,  SDK on ISO/android Implemented it , tingyun will send APP probe data to backend , and it will get a traceId from backend with a request, so it can bind user id with traceId. But now, we consider about to replace it. Is it possbile, we can format probe data with opentrace potocol, send it to skywalking oap, save it with the first span? So we strung the font and back ends together.  I just guess , I don't have any front-end development experience.

**Reply by wu-sheng** [reply]
Yes, you paid them, so, you have that.

**Reply by wu-sheng** [reply]
OT is dead, and it is not a format. Please be sure what you were saying.

**Reply by rexconquer** [reply]
A tools? Whatever it is , I mean I need find a way to send event trace data to skywalking oap from APPs, and strung together with backend trace.Is it possible?

**Reply by wu-sheng** [reply]
It is always possible. Every agent is wrote by someone, some codes.

**Reply by rexconquer** [reply]
Awesome. With the right direction, I can promote it. Thanks again.
